### PR TITLE
Implement shared layer visibility

### PR DESCRIPTION
**Pull request by nbuonin**
Implements the ability for students to see other students layers, BUT
only once those students have already submitted.

### CODE REVIEW

File: locustempus/main/serializers.py [hunk 1]
```diff
@@ -34,12 +35,49 @@ class ProjectSerializer(serializers.ModelSerializer):
...
+        request = self.context['request']
+        user = request.user
+        course = obj.course
+
+        if course.is_true_faculty(user):
+            return [reverse('api-layer-detail', args=[lyr.pk], request=request)
+                    for lyr in obj.layers.all()]
+
+        elif hasattr(obj, 'activity') and course.is_true_member(user):
+            all_layers = [
```
Comment (sdreher) [inline locustempus/main/serializers.py:57]: In the rendered Base Layers view, ideally, the instructor's layers would sort first, then the student layers sorted by last, first / username. I'm not sure if you want to tackle the sorting up here on the server or down at the client.

Comment (zmustapha) [reply locustempus/main/serializers.py:57]: I agree, the Author's layers should be on to at all times, contributors should be of some order as well

File: cypress/integration/activity-response.spec.js [hunk 2]
```diff
@@ -457,6 +439,39 @@ describe('Activity Response Stories', function() {
...
+
+        // Now student-three submitts a response, should now see
+        // student-one's layer
+        // Navigate to the response tab
+        cy.get('[data-cy="Response"]').click();
+        cy.get('[data-cy="submit-or-update-response"]').click();
+
+        cy.reload();
+
+        // Verify Student One's layer is not in Base Layers
```
Comment (sdreher) [inline cypress/integration/activity-response.spec.js:467]: Comment can be updated.

**Review by sdreher — COMMENTED** [review]
Just a minor comment update note. I'll go ahead and merge.

**Pull request merged**